README.md

```diff
@@ -65,6 +67,23 @@ Installation instructions assets-iac-terraform-azure by running:
    > `Depends on Important Note`  
    > Some modules expects that you have resources created. e.g. a Resource Group. Then make sure you include a "depends_on"
 
+4. Execute the modules by running the following [terraform commands](https://www.terraform.io/cli/commands):
+    * terraform init 
+        > Prepare your working directory for other commands
+    * terraform validate
+        > Check whether the configuration is valid
+    * terraform plan
+        > Show changes required by the current configuration
+    * terraform apply
+        > Create or update infrastructure
+
+    > `Azure Authentication Important Note`  
+    > In order to provisioning the resources in Azure using the modules on this repository, you need to be authenticated first. Take a look at those links.
+    > * [Azure Provider: Authenticating using a Service Principal with a Client Certificate](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/guides/service_principal_client_certificate)
```

Comment:
Sometimes creating a Service Principal is not the most straight forward scenario for newbies. In the end, they will end up using `az cli` and do the `az login`. Shouldn´t we just include the `az cli` in the installation and then the command?